starter-code/views/drones/index.ejs

```diff
@@ -0,0 +1,22 @@
+<h2> Drones</h2>
+
+<table class="kill me now table table-hover">
```

Comment:
WHY???!!!

Not, seriously, don't use tables for layout. That said, in this case you can defend the use of tables.